### DISCUSSION TITLE
Ability to Toggle or Add Countdown Timer Clock on Ente Auth

### DISCUSSION THREAD

**Discussion by mallomar** [ente-io/ente#4006] [Enhancements]
Hi – 

Personally I find a countdown timer more helpful instead of a status bar. It would be great to have ti be able to toggle it to show it in lieu of or in addition to the current status bar.

**Comment by Condobloke** [comment]
While I find Ente Auth to be a breath of fresh air to the cluttered world of authorization apps.... I have to admit, I miss the 'clock'
i would not like to take the minds of the people who designed this masterpiece, off their allotted tasks, but if a 'clock' could find its way onto the page, that would be really comforting.

The fact that such a brilliant piece of software is free, is a blessing to us all. 

The rest of the authorization app makers need to hang their heads in shame.
